src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/Import.cs

```diff
@@ -64,9 +62,21 @@ public override IEnumerable<DependencyListEntry> GetStaticDependencies(NodeFacto
             return new DependencyListEntry[] { new DependencyListEntry(ImportSignature, "Signature for ready-to-run fixup import") };
         }
 
-        public int CompareToImpl(ISortableSymbolNode other, CompilerComparer comparer)
+        public override int CompareToImpl(ISortableNode other, CompilerComparer comparer)
         {
-            return new ObjectNodeComparer(comparer).Compare(this, (Import)other);
+            Import otherNode = (Import)other;
+            int result;
+            if (CallSite != null || otherNode.CallSite != null)
```

Comment:
This whole thing with CallSite can be simplified by just calling `String.Compare(CallSite, otherNode.CallSite)`. Accepts null parameters

Reply:
Much cleaner! Changed.